flixel/group/FlxSpriteGroup.hx

```diff
@@ -787,6 +787,18 @@ class FlxTypedSpriteGroup<T:FlxSprite> extends FlxSprite
 			transformChildren(pixelPerfectTransform, Value);
 		return super.set_pixelPerfectRender(Value);
 	}
+	
+	override function set_camera(camera:FlxCamera):FlxCamera
+	{
+		group.set_camera(camera);
```

Comment:
Seems a bit strange to call the setter directly instead of going through the property?

Reply:
Indeed. Normally it's not possible of course as the setter is private. I did it just for symmetry with the other setter calls so it is immediately obvious it was just duplicating the behaviour on the group. I'm not particularly tied to that idea, though. :P